### PR TITLE
MSE should be defined as a mean

### PR DESCRIPTION
**Pull request by cashlo**
The MSE in the first part of 01a is defined as a sum instead of a mean

### CODE REVIEW

**Comment by JFuellem** [comment]
And successively in the gradient descent explanation, where the MSE is written correctly, it probably would have confused me less (having a bit of a rusty math background) if the gradients were also including the factor.
But probably this usually omitted? I only see the benefit that the learning rate could behave more predictably across differently sized sets.

**Comment by cashlo** [comment]
I feel this is probably intentional, but its literally called a mean?
Please close if intentional.
